packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2150,6 +2150,21 @@ export default class Meeting extends StatelessWebexPlugin {
     );
   };
 
+  /**
+  *  Checks if the machine has at least one audio or video device
+  * @param {object} [options]
```

Comment:
Let's use this format for documenting the object properties:

```js
* @param {Object} options
* @param {Boolean} options.sendAudio
* @param {Boolean} options.sendVideo
```

Reply:
Done!